qualang_tools/digital_filters/filters.py

```diff
@@ -52,12 +63,9 @@ def calc_filter_taps(
     if bounce is not None or delay is not None:
         feedforward_taps = bounce_and_delay_correction(bounce, delay, feedforward_taps, Ts)
 
-    max_value = max(np.abs(feedforward_taps))
-
-    if max_value >= 2:
-        feedforward_taps = 1.5 * feedforward_taps / max_value
+    feedforward_taps = _check_hardware_limitation_for_delay(qop_version, feedforward_taps, len(feedback_taps))
```

Comment:
I would probably just combine it with the `_check_hardware_limitation` - Don't see a need for two functions and two calls

Reply:
I split it because I thought about calling the "delay limitation" only for the bounce and delay function, but it won't do anything to the other functions so I can combine them